setup.cfg

```diff
@@ -25,6 +25,7 @@ force_single_line = True
 combine_as_imports = True
 sections = FUTURE,STDLIB,THIRDPARTY,ZOPE,FIRSTPARTY,LOCALFOLDER
 known_third_party = six
+known_zope =
```

Comment:
You don't need to add this if you leave it empty

Reply:
Without the `known_zope` key I get the there the following error:
```
/home/jugmac00/Projects/DocumentTemplate/.tox/flake8/lib/python3.8/site-packages/isort/settings.py:368: UserWarning: `sections` setting includes ZOPE, but no known_zope is defined. The following known_SECTION config options are defined: .
  warn(
```

In order to stay consistent with the other projects I wanted to keep `sections = FUTURE,STDLIB,THIRDPARTY,ZOPE,FIRSTPARTY,LOCALFOLDER` unchanged.

ie if I remove `known_zope` I'd also have to `ZOPE` from `sections`.

Which way do you prefer? Remove both or keep both?

Reply:
> Without the `known_zope` key I get the following error:

OK, that must be a behavior change in `isort`. Leave it in.